### PR TITLE
Add descriptions to all RDUniform methods

### PR DESCRIPTION
**Pull request by DevPoodle**
I added descriptions for all three RDUniform methods.

### CODE REVIEW

File: doc/classes/RDUniform.xml [hunk 1]
```diff
@@ -13,16 +13,19 @@
 			<return type="void" />
 			<param index="0" name="id" type="RID" />
 			<description>
+				Binds the id to the uniform. The data associated with the id is then used when the uniform is passed into a shader.
```
Comment (AThousandShips) [inline doc/classes/RDUniform.xml:16]:
```suggestion
				Binds the given id to the uniform. The data associated with the id is then used when the uniform is passed to a shader.
```
Minor wording nitpick, matter of preference

**Review by AThousandShips — APPROVED** [review]
LGTM otherwise

**Comment by AThousandShips** [comment]
Please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**